runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -82,6 +85,23 @@ RuntimeURLRequestContextGetter::RuntimeURLRequestContextGetter(
 RuntimeURLRequestContextGetter::~RuntimeURLRequestContextGetter() {
 }
 
+inline int GetDiskCacheSize() {
```

Comment:
I think you can drop the `inline` part here, and please move this function upper in the file (to an anonymous namespace either inside or outside the `xwalk` one, and before the `RuntimeURLRequestContextGetter` implementations).